src/classes/PMT_PaymentCreator.cls

```diff
@@ -263,20 +257,13 @@ public without sharing class PMT_PaymentCreator {
                         paymentsToInsert.add(op);                                              
                     }   
                 }
-                //try{
```

Comment:
Trying to remove old commented out code that we don't need anymore when I see it in a file I'm working on.